### DISCUSSION TITLE
Open source sometime?

### DISCUSSION THREAD

**Discussion by pvinis** [MobiusSync/MobiusSync#38] [Q&A] [answered]
I saw in the FAQ that it's closed source for now. Are there plans to open it up later? I feel we could have more people helping (me included) in this app if it becomes open.

If not, totally fine, but I would like some reasons or plans, just to have an idea about the app and the way it will be handled.

**Comment by SimonPickup** [comment, marked as answer]
Justifications for keeping it closed source (for now) are in the FAQ under "Isn't charging for Möbius Sync against the spirit of Syncthing as OSS?". There was some talk on the Syncthing forum and on IRC. Also happy to discuss further here.

Basically it's closed-source to protect the income stream, which incentivizes further development. Some have tried to convince us that it can be open-source GPL and we would protected from somebody publishing a free copycat app in the App Store, but I haven't seen convincing enough evidence or examples of this yet.

There are a few reasons to open-source:
1. To allow people to see that it's not doing anything malicious.
2. To broaden the development resource base.
3. To make sure it doesn't die if I disappear.

I sympathise with all of these but haven't found a way that balances everything appropriately yet.

**Reply by pvinis** [reply]
Mm, gotcha. That makes sense, and I did buy it as soon as I found the button for it. Hm, there must be ways, but yea, until then, I understand and appreciate your work. I hope you/we find a way soon, to keep your income from it and allow for opensourcing.
